alacritty.yml

```diff
@@ -259,6 +259,15 @@ mouse:
   double_click: { threshold: 300 }
   triple_click: { threshold: 300 }
 
+  # URL Launcher
+  #
+  # This program is executed when clicking on a text which is recognized as a URL.
+  # The URL is always added to the command as the last parameter.
+  #url_launcher:
+  #  program: /usr/bin/firefox
```

Comment:
Yeah, I've thought about that and it would probably be a good idea. I just don't have it installed to test it, but I'll give it a go to see which parameters might be required.